clients/java/signalr/src/main/java/com/microsoft/signalr/OkHttpWebSocketWrapper.java

```diff
@@ -19,17 +22,17 @@
     private String url;
     private Map<String, String> headers;
     private OkHttpClient client;
-    private Logger logger;
     private OnReceiveCallBack onReceive;
     private BiConsumer<Integer, String> onClose;
     private CompletableFuture<Void> startFuture = new CompletableFuture<>();
     private CompletableFuture<Void> closeFuture = new CompletableFuture<>();
 
-    public OkHttpWebSocketWrapper(String url, Map<String, String> headers, OkHttpClient client, Logger logger) {
+    private final Logger logger = LoggerFactory.getLogger(OkHttpWebSocketWrapper.class);
```

Comment:
Not a fan of using this class as the "category" but I won't stop the PR because of it.